lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -759,6 +832,17 @@ def cmd_info_tabs(str, words)
     cmd_use_tabs(str, words)
   end
 
+  #
+  # Tab completion for the options command (same as use)
+  #
+  # @param str [String] the string currently being typed before tab was hit
```

Comment:
Both of the `@param` lines can be replaced with a single line like this: `@param (see #cmd_use_tabs)`